streams/src/main/java/org/apache/kafka/streams/processor/internals/Task.java

```diff
@@ -97,13 +97,9 @@ public boolean isValidTransition(final State newState) {
         }
     }
 
-    TaskId id();
```

Comment:
Group and order methods (compare in-line comments). -- Sub-classed inherit this ordering.